.github/ISSUE_TEMPLATE/V&V_request.md

```diff
@@ -62,26 +66,32 @@ Note: not applicable items will be struck through
 
 *Ensure all above items have been completed before continuing*
 
-@biosensics/test-and-manufacturing :
+**@biosensics/test-and-manufacturing :**
 - [ ] V&V executed
 - [ ] Completed V&V submitted to [Quality](mailto:quality@biosensics.com)
 
-@biosensics/engineering :
-- [ ] App ECO Section II submitted to [Quality](mailto:quality@biosensics.com)
-- [ ] App ECO Section II approved
+**@biosensics/engineering :**
+
+Server:
 - [ ] Server ECO Section II submitted to [Quality](mailto:quality@biosensics.com)
 - [ ] Server ECO Section II approved
-- [ ] Ensure correct Zoom meeting room is set
 - [ ] Create server release
 - [ ] Deploy server release to Aptible production server
 - [ ] Confirm any updated Aptible production server configuration values
-- [ ] Merge `release` branch into `master` on server repository
+- [ ] Merge `release` branch into `develop` on server repository
+
+Application:
+- [ ] App ECO Section II submitted to [Quality](mailto:quality@biosensics.com)
+- [ ] App ECO Section II approved
 - [ ] Merge `release` branch into `master` on Application repository
 - [ ] Send release APK to operations and quality for production use
 
 
-@biosensics/quality :
+**@biosensics/quality :**
 - [ ] App ECO closed & filed to QSR
 - [ ] Server ECO closed & filed to QSR
 - [ ] Insert file path(s) to documentation (ECO, V&V etc):
 - [ ] New version released
+
+**@biosensics/operations :**
+- [ ] Confirm zoom room on release build
```

Comment:
Maybe Telehealth instead of Zoom as we support Zoom and Twilio.

Reply:
thats fine with me, ill make that change